.github/workflows/workflow-java.yaml

```diff
@@ -34,15 +31,6 @@ on:
         type: string
         default: "false"
         description: "Build and push a staging tagged container image for this application on commits to the mainline_branch."
-      do_production_build:
-        type: string
-        default: "true"
-        description: "Build and push a production tagged container image for this application on commits to the mainline_branch."
-
-concurrency:
-  group: ${{ github.ref }}
```

Comment:
FYI `group: ${{github.ref}}` is really problematic because it doesn't include something like the workflow name, so it literally means only 1 job can run per ref at a time.